### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by awright18**
As part of the small task to [create issue templates in public repos](https://github.com/Particular/CustomerSuccess/issues/2401), this PR adds 4 files to the repos
- Template for bugs
image
- Template for improvements
image
- Template for features
image
- Support file that links to our support page
When opening an issue there is a link to "support" 
image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Report a bug.
```
Comment (soujay) [inline .github/ISSUE_TEMPLATE/bug_report.yml:3]:
```suggestion
description: Report a bug.
```

Comment (soujay) [reply .github/ISSUE_TEMPLATE/bug_report.yml:3]: As per this [doc ](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository), yml  files should have  a description.

**Comment by adamralph** [comment]
> this PR adds 3 files to the repos

I see 6 files.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: textarea
+    id: additional-information
+    attributes:
+      label: Additional Information
+      description: If there are any possible solutions, workarounds, or additional information please describe here
```
Comment (HindyLambrozo) [inline .github/ISSUE_TEMPLATE/bug_report.yml:39]:
```suggestion
      description: If there are any possible solutions, workarounds, or additional information please describe them here
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+name: Feature Request
+description: Request a new feature.
+labels: ["Feature" ]
+body:
+  - type: markdown
+    attributes:
+      value: | 
+        Please check the open and closed issues to see if the feature has already been requested.
```
Comment (HindyLambrozo) [inline .github/ISSUE_TEMPLATE/feature_request.yml:8]:
```suggestion
        Please check all open and closed issues to see if the feature has already been requested.

```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the new requested feature.
+      description: A clear and concise description of what the feature is.    
+      value: | 
+        **Is your feature related to a problem? Please describe.**
+        
+        **Describe the requeste feature**
```
Comment (HindyLambrozo) [inline .github/ISSUE_TEMPLATE/feature_request.yml:17]:
```suggestion
        **Describe the requested feature**

```

File: .github/ISSUE_TEMPLATE/improvement_request.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+name: Improvement Request
+description: Suggest an improvement to an existing code base.
+labels: ["Improvement" ]
+body:
+  - type: markdown
+    attributes:
+      value: | 
+        Please check the open and closed issues to see if improvement has already been suggested.
```
Comment (HindyLambrozo) [inline .github/ISSUE_TEMPLATE/improvement_request.yml:8]:
```suggestion
        Please check all open and closed issues to see if improvement has already been suggested.
```

**Review by HindyLambrozo — APPROVED** [review]
editorial review

File: .github/SUPPORT.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# Looking for Support
+
+Check out our support options at [https://particular.net/support](https://particular.net/support)
```
Comment (adamralph) [inline .github/SUPPORT.md:3]:
```suggestion
Check out our [support options](https://particular.net/support).
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |     
+        For additional support options visit [particular.net/support](https://particular.net/support)
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Describe the bug 
+      description: A clear and concise description of what the bug is.
```
Comment (adamralph) [inline .github/ISSUE_TEMPLATE/bug_report.yml:13]:
```suggestion
      description: A clear and concise description of the bug.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+    id: what-happened
+    attributes:
+      label: Describe the bug 
+      description: A clear and concise description of what the bug is.    
+      value: | 
+        **Description**
+        
+        **Expected behavior**
+                
+        **Versions** - Please list the version of the relevant packages or applications in which the bug exists.
```
Comment (adamralph) [inline .github/ISSUE_TEMPLATE/bug_report.yml:19]:
```suggestion
        ## Description
        
        ## Expected behavior
        
        ## Actual behavior
        
        ## Versions
        
        Please list the version of the relevant packages or applications in which the bug exists.        
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+        **Expected behavior**
+                
+        **Versions** - Please list the version of the relevant packages or applications in which the bug exists.        
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to reproduce
+      description: Detailed instructions on how to reproduce the bug.
```
Comment (adamralph) [inline .github/ISSUE_TEMPLATE/bug_report.yml:26]:
```suggestion
      description: Detailed instructions to reproduce the bug.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+    id: additional-information
+    attributes:
+      label: Additional Information
+      description: If there are any possible solutions, workarounds, or additional information please describe here
+      value: | 
+       **Workarounds** 
+       
+       **Possible Solutions**
+       
+       **Additional Information**
```
Comment (adamralph) [inline .github/ISSUE_TEMPLATE/bug_report.yml:45]:
```suggestion
       ## Workarounds
       
       ## Possible solutions
       
       ## Additional information
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 11]
```diff
@@ -0,0 +1,33 @@
+name: Feature Request
```
Comment (adamralph) [inline .github/ISSUE_TEMPLATE/feature_request.yml:1]:
```suggestion
name: Feature request

```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+labels: ["Feature" ]
+body:
+  - type: markdown
+    attributes:
+      value: | 
+        Please check the open and closed issues to see if the feature has already been requested.
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the new requested feature.
```
Comment (adamralph) [inline .github/ISSUE_TEMPLATE/feature_request.yml:12]:
```suggestion
      label: Describe the feature.

```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+      description: A clear and concise description of what the feature is.
```
Comment (adamralph) [inline .github/ISSUE_TEMPLATE/feature_request.yml:13]:
```suggestion
      description: A clear and concise description of the feature.

```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+    id: description
+    attributes:
+      label: Describe the new requested feature.
+      description: A clear and concise description of what the feature is.    
+      value: | 
+        **Is your feature related to a problem? Please describe.**
+        
+        **Describe the requeste feature**
+                
+        **Describe alternatives you've considered**
```
Comment (adamralph) [inline .github/ISSUE_TEMPLATE/feature_request.yml:19]:
```suggestion
        ## Related problem
        
        <!-- Is your feature related to a problem? If please describe the problem.-->
        
        ## Feature description
                
        ## Alternatives considered

```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: additional-context
+    attributes:
+      label: Additional Context
+      description: Add any other context about the request here.
```
Comment (adamralph) [inline .github/ISSUE_TEMPLATE/feature_request.yml:26]:
```suggestion
      description: Add any other context about the request.

```

File: .github/ISSUE_TEMPLATE/improvement_request.yml [hunk 16]
```diff
@@ -0,0 +1,33 @@
+name: Improvement Request
```
Comment (adamralph) [inline .github/ISSUE_TEMPLATE/improvement_request.yml:1]:
```suggestion
name: Improvement request
```

**Comment by soujay** [comment]
> As a customer, I'm not sure I'd know when to use "Feature request" vs "Improvement request". I think we should consider omitting "Improvement request".

Some repos have separate labels for feature and improvement. Hence, we decided it's best to have distinct templates for both as a first step. The templates could be improved upon on further iterations if we received any feedback from users.

**Pull request merged**